hosts/alcyone/services/gelos-website.nix

```
@@ -53,6 +53,9 @@ in {
         "=/if4".return = "301 https://gelos.club/2023/08/21/installfest-2023-2.html";
         "=/installfest-5".return = "301 https://gelos.club/2024/03/01/installfest-2024-1.html";
         "=/if5".return = "301 https://gelos.club/2024/03/01/installfest-2024-1.html";
+        "=/installfest-6".return = "301 https://gelos.club/2025/03/25/installfest-2025-1.html";
+        "=/if6".return = "301 https://gelos.club/2025/03/25/installfest-2025-1.html";
+        "=/AulaShell2025".return = "301 https://gelos.club/2025/03/24/aula-shell.html";
```

Os matches são case sensitive por padrao. Tem certeza que é com esse casing?

Aqui pra deixar case insensitive:
```suggestion
        "~* ^/AulaShell2025$".return = "301 https://gelos.club/2025/03/24/aula-shell.html";
```

---

A casing era essa msm pq foi como eu botei no flyer (fica mais fácil de ler do q ser tudo minúsculo), mas se da pra deixar case insensitive melhor ainda então 

Bota insensitivity lá no https://github.com/gelos-icmc/infra/pull/32 tbm pfv